softdev.bib

```diff
@@ -3342,3 +3342,11 @@ @inproceedings{cheng20typefreezing
     month = Feb,
     booktitle = "CGO"
 }
+
+@inproceedings{brunthaler10quickening,
+    title = "Efficient Interpretation using Quickening",
+    author = "Brunthaler, Stefan",
+    year = 2010,
+    month = Oct,
+    booktitle = {DLS}
```

Comment:
Is there a DOI for this? If so, it's good to add it, as some LaTeX styles make the links clickable.

Reply:
Added :point_right: b8b60873b93b5b40cbd181a4edb8c26d25a6f60c